x-pack/plugins/security_solution/public/timelines/components/timeline/data_providers/index.tsx

```diff
@@ -84,7 +89,36 @@ const getDroppableId = (id: string): string =>
  * the user to drop anything with a facet count into
  * the data pro section.
  */
+
+const timelineSelectModeItemsClassName = 'timelineSelectModeItemsClassName';
+
+const searchOrFilterPopoverClassName = 'searchOrFilterPopover';
+const searchOrFilterPopoverWidth = '352px';
+
+const SearchOrFilterGlobalStyle = createGlobalStyle`
```

Comment:
Why do we need to use global styles? can't this rules be nested inside the `CustomTooltipDiv` styles?

Reply:
@semd, this is needed because they style a popover which is inserted into a data-portal which directly a child of `body`

Reply:
Would it be possible to pass the `css` prop with the `popoverProps`? Using global styles has caused complex bugs in the past, we should avoid them. 
If the `css` in `popoverProps` works we should also be able to remove all those `!important`, which we should also try to avoid.

Reply:
This is taking more time that I thought, there some with super select passing props to popover. still working on it.. But Other changes are in.. I will push this as separate commit.

Reply:
Resolved here : bfc124625b022d4edde8c2227dd243db97976555

Reply:
Thanks!